Dockerfile

```diff
@@ -128,23 +127,6 @@ RUN \
     make -j ${NPROC} install; \
     cd ..; rm -rf netcdf
 
-# WEBP
```

Comment:
Why not build webp and zstd? Should only be very small and maybe someone wants to use them. Since this is meant to be a generally usable fork of the original `geolambda`, simply built with python instead of plain lambda base, I would leave everything in.